scala/tempo/src/main/scala/com/databrickslabs/tempo/io.scala

```diff
@@ -0,0 +1,59 @@
+package com.databrickslabs.tempo
+
+import org.apache.spark.sql.functions._
+import org.apache.spark.sql.SparkSession
+
+
+/**
+The following methods provide support for writing the TSDF object to Delta Lake based on the partition columns specified along with the time as ZORDER columns (with Hilbert space-filling curve optimization)
+ */
+object TSDFWriters {
+  /**
+    * param: tsdf: input TSDF object to write
+    * param: tabName Delta output table name
+    * param: optimizationCols list of columns to optimize on (time)
+    */
+  def write(tsdf: TSDF, spark: SparkSession, tabName: String, optimizationCols: Option[Seq[String]] = None): Unit = {
+
+    // hilbert curves more evenly distribute performance for querying multiple columns for Delta tables
+    spark.conf.set("spark.databricks.io.skipping.mdc.curve", "hilbert")
+
+    val df = tsdf.df
+    val ts_col = tsdf.tsColumn.name
+    val partitionCols = tsdf.partitionCols
+
+    var optCols = Seq("")
+
+    optimizationCols match {
+      case Some(oc) => {
+        optCols = oc ++ Seq("event_time")
+      }
+      case None => {
+         optCols = Seq("event_time")
+      }
+    }
+
+    var useDeltaOpt = true
+
+    try {
+      val ddf = spark.range(10).write.mode("overwrite").partitionBy("event_dt").format("delta").saveAsTable("ddft")
```

Comment:
Is this the best way to confirm this? I think there are some spark params to tell you the driver mode. Either way, since you're saving a table, it should also be dropped after to not leave crumbs behind.